src/ui_interface.cpp

```diff
@@ -35,6 +35,7 @@ struct UISignals {
     }
 
 ADD_SIGNALS_IMPL_WRAPPER(ThreadSafeMessageBox);
+ADD_SIGNALS_IMPL_WRAPPER(ThreadSafeMessageBox2);
```

Comment:
Please use a better name than `ThreadSafeMessageBox2`. What makes this function different?

Reply:
Fixed.

btw, `2` in `ThreadSafeMessageBox2` meant "bi-" in **bi**lingual rather "second" :)